variables.tf

```diff
@@ -0,0 +1,5 @@
+variable "source_documents" {
+  type        = "list"
+  description = "List of JSON IAM policy documents (List size limited to 10)"
```

Comment:
Add a note about sensitivity to order of items in list and how they will override each other